doc/mkdocs/docs/api/macros/json_assert.md

```diff
@@ -1,11 +1,85 @@
-# JSON_ASSERT(x)
+# JSON_ASSERT
 
 ```cpp
-JSON_ASSERT(x)
+#define JSON_ASSERT(x) /* value */
 ```
 
-## Default implementation
+This macro controls which code is executed for [runtime assertions](../../features/assertions.md) of the libraries. It
+is set by the library, but can be overridden by defining it before including the library.
```

Comment:
"This macro controls which code to execute for [runtime assertions](../../features/assertions.md) of the library."

At least change "libraries" to "library".

And then to avoid "library" 3 times in a row:

"To override the default definition, define it before including the library."

(The default definition is less than 10 lines below.)

Reply:
I think I don't even need to explain how to override it - JSON_ASSERT was the first macro I documented, and I had to find a style first.